docs/docs/migration-guide.mdx

```diff
@@ -10,6 +10,51 @@ description: |
 This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
+## Rasa 2.4 to 2.5
+
+### Machine Learning Components
+
+#### SpaCy 3.0
+
+Rasa now supports spaCy 3.0. This means that we can support more features for more
+languages but this also introduced a breaking change. SpaCy 3.0 deprecated the
+`spacy link <language model>` command so that means that from now on
+[the full model name](https://spacy.io/models) needs to be used in the `config.yml` file.
+
+**Before**
+
+Before you could run `spacy link en en_core_web_md` and then we would be able
+to pick up the correct model from the `language` parameter.
+
+```yaml
+language: en
+
+pipeline:
+   - name: SpacyNLP
+```
+
+**Now**
+
+This behavior will be deprecated and instead you'll want to be explicit in `config.yml`.
+
+```yaml
+language: en
+
+pipeline:
+   - name: SpacyNLP
+     model: en_core_web_md
+```
+
+**Fallback**
+
+To make the transition easier, Rasa will try to fall back to a medium spaCy model when-ever
+a compatible language is configured for the entire pipeline in `config.yml` even if you don't
+specify a `model`. This fallback behavior is temporary and will be deprecated in spaCy 3.0.
```

Comment:
spacy spacy spacy spacy
```suggestion
specify a `model`. This fallback behavior is temporary and will be deprecated in Rasa Open Source 3.0.0.
```

Reply:
@koaning I don't think this addressed yet based on the current diff.

Reply:
Confirmed now. https://github.com/RasaHQ/rasa/pull/7869/files#diff-144c9993b9753bf4e93513f5c220747c493570dd472eefc19e701791e0da8622R52